docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Actually, the multi-resource example above would also need a special transformation pass to do the magic defaulting. It mainly differs from generators discussed above in that it wouldn't have a specific schema, other than just an ordered list of related resources.